sdcm/sct_events/events_device.py

```diff
@@ -152,7 +152,7 @@ def outbound_events(self,
         with suppress_interrupt():
             for events_counter.value, obj in enumerate(self.inbound_events(stop_event=stop_event), start=1):
                 for filter_key, filter_obj in list(filters.items()):
-                    if filter_obj.expire_time and filter_obj.expire_time < obj.timestamp:
+                    if filter_obj.expire_time and filter_obj.expire_time + 600 < obj.effective_timestamp:
```

Comment:
Shouldn't we make value "600" configurable or more obvious to be changed? Like we do it on lines 33-41 in this module...

Reply:
fixed